src/assets/sass/pandora/base/_grid.scss

```diff
@@ -0,0 +1,195 @@
+app-executions-grid-data {
+  display: none;
+}
```

Comment:
This is a bit weird, even unnecessary maybe?, and different from the way it is done on the dashboard. You didn't need a display:none there. I think, it's because it takes 3fr. Can you set the "grid-template-columns" in a different way, eg compare with the dashboard?

Reply:
Removed and replaced with an "auto" column